testsuite/src/commonMain/kotlin/org/jetbrains/litmuskt/tests/UnsafePublication.kt

```diff
@@ -66,19 +66,42 @@ object UnsafePublication {
         }
     }
 
-    val PlainArray = litmusTest({
+    val PlainArrayBoxed = litmusTest({
         object : LitmusIOutcome() {
             var arr: Array<Int>? = null
         }
     }) {
         thread {
-            arr = Array(10) { 0 }
+            arr = Array(10) { 1 }
         }
         thread {
             r1 = arr?.get(0) ?: -1
         }
         spec {
-            accept(0)
+            accept(1)
+            interesting(0)
```

Comment:
Techinically, should be `NullPointerException` instead, but I guess the tool does not support it yet. 
Maybe worth to leave a comment about it here.